Dockerfile

```diff
@@ -1,20 +1,23 @@
 FROM node:10.15.3-alpine
 
 # Create app directory
-WORKDIR /mine-support-user-service
+WORKDIR /usr/src/app
 
 # Install app dependencies
 # A wildcard is used to ensure both package.json AND package-lock.json are copied
 # where available (npm@5+)
-COPY package*.json ./
+COPY  --chown=node:node package*.json ./
 
 RUN npm install
 RUN npm install -g sequelize-cli
 # If you are building your code for production
 # RUN npm ci --only=production
 
 # Bundle app source
-COPY . .
+COPY --chown=node:node . .
+USER node
 
-EXPOSE 3002
+ARG PORT=3002
+ENV PORT ${PORT}
+EXPOSE ${PORT} 9229 9230

```

Comment:
Is there a risk here in exposing debug ports to other containers in production?

Reply:
I made minimal changes to the docker to get it running in the pipeline, there is a separate card to fix the docker file.

However, these ports would only be active if a container was running with the debug flag, which should not occur in production

Reply:
Found the other card, which already includes making exposed ports configurable so that's good: https://eaflood.atlassian.net/browse/FPD-283